wire-tests-swift/no-manifest/src/main/swift/SwiftEdgeCases.swift

```diff
@@ -6,7 +6,7 @@ public struct SwiftEdgeCases {
 
     @ProtoDefaulted
     public var `return`: String?
-    public var error: SwiftEdgeCases.Error?
+    public var error: SwiftEdgeCases.Error_?
```

Comment:
@lickel the fact that I consolidated this under a single helper method it now affected enums just like it did messages, does it look good to you as a change?

Reply:
it also looks like enums called `Type` are also emitted `Type_` so this looks more aligned now

Reply:
Nice